parts/kubernetesmastervars.t

```diff
@@ -34,8 +34,9 @@
     "kubernetesKubeDNSSpec": "[parameters('kubernetesKubeDNSSpec')]",
     "kubernetesDNSMasqSpec": "[parameters('kubernetesDNSMasqSpec')]",
     "networkPolicy": "[parameters('networkPolicy')]",
-    "azureVnetCniURL":"[parameters('azureVnetCniURL')]",
-    "azureCniURL":"[parameters('azureCniURL')]",
+    "cniPluginsURL":"[parameters('cniPluginsURL')]",
+    "vnetCniLinuxPluginsURL":"[parameters('vnetCniLinuxPluginsURL')]",
+    "vnetCniWindowsPluginsURL":"[parameters('vnetCniWindowsPluginsURL')]",
```

Comment:
@ofiliz Where are we actually using the `vnetCniWindowsPluginsURL` variable? Mainly: how are we implementing the conditional CNI URL whether or not the platform in Linux or Windows?

Reply:
@jackfrancis I am using the Windows plugin URL in a separate PR coming next week. There won't be a conditional check anywhere, the kubernetesmastercustomscript.sh will use the Linux version and the kuberneteswindowssetup.ps1 scripts will use the Windows version.

Because we are already renaming these, I thought I'll add the correct "Linux" vs. "Windows" qualifiers to the names now, to prevent further churn next week.

Reply:
Understood, so the Windows artifacts will be unused but referable with a follow-up PR. 👍